.github/workflows/cmake.yml

```
@@ -587,6 +587,7 @@ jobs:
             shell: msys2 {0}
             compiler: i686-w64-mingw32-gcc
             cxx-compiler: i686-w64-mingw32-g++
+            gcov-exec: ${{ runner.temp }}/msys64/mingw32/bin/gcov
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)